index.js

```diff
@@ -21,24 +21,24 @@ function buildTargets(options) {
   return assign({}, defaultTargets, options.additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
+module.exports = function buildAirbnbPreset(api, options) {
   var transpileTargets = (options && options.targets) || buildTargets(options || {});
 
   var debug = (options && typeof options.debug === 'boolean') ? !!options.debug : false;
 
   return {
     presets: [
-      require('babel-preset-env').default(null, {
+      require('@babel/preset-env').default(api, {
         debug: debug,
         exclude: [
           'transform-async-to-generator',
-          'transform-es2015-template-literals',
+          'transform-template-literals',
           'transform-regenerator'
         ],
         modules: false,
         targets: transpileTargets
       }),
-      require('babel-preset-react')
+      [require('@babel/preset-react'), { development: api.env() === 'development' }]
```

Comment:
Better to use `api.env('development')`. Explanation in http://babeljs.io/docs/en/config-files#apienv

Reply:
Fixed